### PR TITLE
Skip worker_bootstrap when cloud_framework is openstack, bsc#1089652

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Comment by mjura** [comment]
Travis is complaining
```
Offenses:
app/controllers/setup_controller.rb:67:3: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for worker_bootstrap is too high. [8/6]
  def worker_bootstrap
  ^^^
app/controllers/setup_controller.rb:67:3: C: Metrics/PerceivedComplexity: Perceived complexity for worker_bootstrap is too high. [8/7]
  def worker_bootstrap
  ^^^
166 files inspected, 2 offenses detected
```
Don't know if should I fix it or we are fine?

**Comment by kiall** [comment]
> Don't know if should I fix it or we are fine?

Ehh - Lets add a ignore comment for now to keep travis happy. We want to merge ASAP.

File: app/controllers/setup_controller.rb [hunk 1]
```diff
@@ -67,7 +67,8 @@ def configure
   def worker_bootstrap
     @controller_node = Pillar.value pillar: :dashboard
```
Comment (MaximilianMeister) [inline app/controllers/setup_controller.rb]: just add this around the method definidion:
```ruby
# rubocop:disable Metrics/CyclomaticComplexity
def worker_bootstrap
  ...
end
# rubocop:enable Metrics/CyclomaticComplexity
```

Comment (mjura) [reply app/controllers/setup_controller.rb]: done

**Comment by kiall** [comment]
> codeclimate can be ignored i guess +1

That's a bug in GitHub! (really .. theres something horribly wrong with GitHub the last few days ;))

**Comment by mjura** [comment]
[Bug 1089652 - Can't bootstrap cluster due to missing /srv/velum/config/data/openstack-types.json in the velum image](https://trello.com/c/uSXx4frn/793-bug-1089652-cant-bootstrap-cluster-due-to-missing-srv-velum-config-data-openstack-typesjson-in-the-velum-image)

**Pull request merged**